cypress/integration/ms-stream-video.spec.js

```diff
@@ -0,0 +1,13 @@
+/// <reference types="cypress" />
+
+context('<MsStreamVideo />', () => {
+  it('it loads stream video correctly', () => {
+    cy.visit('/iframe.html?id=components-msstreamvideo--usage&viewMode=story');
+    cy.getIframeBody().then((data) => {
+      console.log({data});
+    });
+    cy.getIframeBody().find('#streamLoading');
```

Comment:
I removed the `.should('not.be.undefined');` used in other components' tests.
It seems to make tests pass when they shouldn't whereas simply stopping after the `find()` works well.

For an example, I modified the youtube tests on this branch: 
https://github.com/gaelhameon/mdx-embed/blob/wip/questions/cypress/integration/youtube.spec.js